helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -128,7 +128,15 @@ rules:
       - configmaps
     verbs:
       - get
-      - get
+      - create
```

Comment:
I think we should not create the promxy configmap. It should be a guarantee that promxy configmap is present.

That's the only interface we should agree on: having a configmap in a given namespace with a given name.

Reply:
I agree, but can we do that with our current tooling as promxy app is installed using the app-chart mechanism?